plugins/inputs/opentelemetry/opentelemetry.go

```diff
@@ -105,17 +127,17 @@ func (o *OpenTelemetry) Stop() {
 	if o.grpcServer != nil {
 		o.grpcServer.Stop()
 	}
+	o.listener = nil
 
 	o.wg.Wait()
 }
 
 func init() {
 	inputs.Add("opentelemetry", func() telegraf.Input {
 		return &OpenTelemetry{
-			ServiceAddress:      "0.0.0.0:4317",
 			SpanDimensions:      otel2influx.DefaultOtelTracesToLineProtocolConfig().SpanDimensions,
 			LogRecordDimensions: otel2influx.DefaultOtelLogsToLineProtocolConfig().LogRecordDimensions,
-			MetricsSchema:       "prometheus-v1",
```

Comment:
This is a very helpful default value. Why remove it?

Reply:
Because we usually handle those in the `Init()` function, so it is in fact not removed.